pavelib/tests.py

```diff
@@ -31,11 +31,15 @@
     ("fail-fast", "x", "Fail suite on first failed test"),
     ("fasttest", "a", "Run without collectstatic"),
     make_option(
+        "--eval-attr", dest="eval_attr",
+        help="Only run tests matching given attribute expression."
+    ),
```

Comment:
Ah - so this arg is the way to check for test tags, such as whitelist/blacklist?

Reply:
Correct, and it supports boolean expressions like "[not shard and not a11y](https://github.com/edx/edx-platform/blob/a8c7ecdbe5ba2f253e106f45e7792dc8ba47f57b/scripts/generic-ci-tests.sh#L182)"